backend/prisma/schema.prisma

```diff
@@ -146,7 +148,7 @@ model attendances {
   id            Int       @id @default(autoincrement())
   course_id     Int       @map("course_id")
   lecture_id    Int       @map("lecture_id")
-  user_id       Int       @map("user_id")
+  user_id       Int
```

Comment:
To keep consistency with other column definitions, let's keep the @map declaration.